doc/configuration.rst

```diff
@@ -481,18 +481,24 @@ We recommend that you copy this set of exit codes to your ``luigi.cfg`` file:
   missing_data=20
   task_failed=30
   scheduling_error=35
+  unknown_reason=38 or 0
   unhandled_exception=40
 
 unhandled_exception
   For internal Luigi errors.  Defaults to 4, since this type of error
   probably will not recover over time.
+unknown_reason
+  For when a task does not run successfully because of an unknown reason. Despite
+  this case can be expected in an error free execution, it does not guarantee
+  completeness of the root task. Return code 38 is advised only for cases in which
+  Luigi return code 0 is used by the user to guarantee root tasks' completeness.
```

Comment:
Missed the [comment suggesting "38 or 0"](https://github.com/spotify/luigi/pull/1771#discussion_r71119648). The comment doesn't describe the benefit. I would -1 this convoluted wording as overcomplication.

0 is already the default value. The existing users who perform retcode configuration won't be taken by surprise.